Sources/PT.PM.Common/PathUtils.cs

```diff
@@ -37,11 +36,14 @@ private static string NormalizePath(this string path, bool isDirectory = true, b
 
         public static string NormalizeDirSeparator(this string path)
         {
-            string notPlatformSeparator = CommonUtils.IsWindows ? "/" : "\\";
+            char notPlatformSeparator = CommonUtils.IsWindows ? '/' : '\\';
 
-            if (path.Contains(notPlatformSeparator))
+            foreach (char c in path)
             {
-                return path.Replace(notPlatformSeparator, Path.DirectorySeparatorChar.ToString());
+                if (c == notPlatformSeparator)
+                {
+                    return path.Replace(notPlatformSeparator, Path.DirectorySeparatorChar);
```

Comment:
Can be replaced with `path.Replace(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar)`. this will help remove the foreach loop

Reply:
My motivation: do not do excess string allocations. But maybe you are right and it does not make sense.